cli/fabric.c

```diff
@@ -729,8 +729,8 @@ static int hvd_detail_body_print(struct switchtec_dev *dev,
 	char bdf_str1[32];
 	char bdf_str2[32];
 	char vep_type_str[32];
-	uint64_t enable_bitmap;
-	uint64_t bitmap;
+	unsigned long long enable_bitmap;
+	unsigned long long bitmap;
```

Comment:
I kind of feel uint64_t is still the better choice here... You should be able to use the PRIu64 macro to obtain the correct format specifier. But this is also an unrelated bugfix for a third patch.